packages/ckbtc/src/minter.canister.spec.ts

```diff
@@ -194,8 +194,9 @@ describe("ckBTC minter canister", () => {
           NoNewUtxos: {
             required_confirmations: 123,
             current_confirmations: toNullable(456),
+            pending_utxos: [],
           },
-        },
+        } as UpdateBalanceError,
```

Comment:
Is this needed?

Reply:
Either this or `as []` after `pending_utxos: []`.
This seemed more future proof.